app/services/cfe/obtain_assessment_result_service.rb

```diff
@@ -0,0 +1,35 @@
+module CFE
+  class ObtainAssessmentResultService < BaseService
+    private
+
+    def cfe_url_path
+      "/assessments/#{@submission.assessment_id}"
+    end
+
+    def query_cfe_service
+      make_get_request
+    end
+
+    def make_get_request
+      conn.get cfe_url_path
+    end
```

Comment:
Would it be simpler to just:
```ruby
def query_cfe_service
  conn.get cfe_url_path
end
```

Reply:
Yep.  I guess there was a need to that once which disappeared with the refactoring.